website/versioned_docs/version-v0.6.x/custom-model.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Custom Model Integration
+---
+
+## Option 1: Use Pre-Built Docker Image Without Model Weights
+If you want to avoid building a Docker image with model weights, use our pre-built reference image (`ghcr.io/kaito-project/kaito/llm-reference-preset:latest`). This image, built with [Dockerfile.reference](../../examples/custom-model-integration/Dockerfile.reference), dynamically downloads model weights from HuggingFace at runtime, reducing the need to create and maintain custom images.
```

Comment:
We need to replace all relative path from `../..` to `../../..` due to the new folder we introduced